dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -126,4 +141,25 @@ private static Options configureOptions() {
       throw new ParseException("Missing argument");
     }
   }
+
+  static void configureMavenRepositories(CommandLine commandLine) throws ParseException {
+    if (!commandLine.hasOption("m")) {
+      return;
+    }
+
+    ImmutableList.Builder<RemoteRepository> repositoryListBulder = ImmutableList.builder();
+    for (String mavenRepositoryUrl : commandLine.getOptionValues("m")) {
+      RemoteRepository repository =
+          new RemoteRepository.Builder(mavenRepositoryUrl, "default", mavenRepositoryUrl).build();
+      if (!ALLOWED_REPOSITORY_URL_PROTOCOL.contains(repository.getProtocol())) {
```

Comment:
there are other ways this could fail. E.g. the input could be "_http_file__https" 

You should probably try to build a java.net.URI object and look for URISyntaxExceptions. You might need to require an absolute URL too.

Reply:
RemoteRepository parses the URL string. For  ‘_http_file__https’, getProtocol method should return an empty string.


I will add the URISyntaxException check.

Reply:
Checking absolute is unnecessary as line 157 already checks the presence of the schema (protocol)

https://docs.oracle.com/javase/8/docs/api/java/net/URI.html#isAbsolute--